prow/jobs/incubator/documentation-component/documentation-component.yaml

```diff
@@ -0,0 +1,37 @@
+job_template: &job_template
+  skip_report: false
+  always_run: true
+  decorate: true
+  path_alias: github.com/kyma-incubator/documentation-component
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181130-b28250b
```

Comment:
As we have agreed to use yarn, and not node, we should make new buildpack with yarn and set it to use it here

Reply:
@aerfio  So please prepare new buildpack

Reply:
v20190724-d41df0f

Reply:
```suggestion
      - image: eu.gcr.io/kyma-project/test-infra/buildpack-node:v20190724-d41df0f
```